src/D2L.Bmx/UpdateChecker.cs

```diff
@@ -1,34 +1,36 @@
+using System.Diagnostics.CodeAnalysis;
 using System.Reflection;
 using System.Runtime.InteropServices;
 using System.Text.Json;
 using D2L.Bmx.GitHub;
 
 namespace D2L.Bmx;
 
-internal class UpdateChecker( IGitHubClient github, BmxConfig config ) {
+internal class UpdateChecker( IGitHubClient github ) {
 	public async Task CheckForUpdatesAsync() {
 		try {
-			var cachedVersion = GetUpdateCheckCache();
```

Comment:
The name `cachedVersion` suggests a `Version` object or a string, but it's actually an `UpdateCheckCache`...